AnkiDroid/src/main/java/com/ichi2/anki/AnkiDroidApp.kt

```diff
@@ -134,7 +135,14 @@ open class AnkiDroidApp : Application() {
             UsageAnalytics.setDryRun(true)
         }
 
-        Timber.i(DebugInfoService.getDebugInfo(this))
+        applicationScope.launch {
+            try {
+                val debugInfo = DebugInfoService.getDebugInfo(this@AnkiDroidApp)
+                Timber.i(debugInfo)
+            } catch (e: Exception) {
+                Timber.w(e, "Error getting debug info")
+            }
```

Comment:
getDebugInfo should not throw

```suggestion
            Timber.i(DebugInfoService.getDebugInfo(this))
```